config/config.exs

```diff
@@ -89,6 +89,7 @@ config :skate,
       }
     ]
   },
+  version: System.get_env("SENTRY_RELEASE"),
```

Comment:
suggestion: I think that the version endpoint would error when [`send_resp/3`](https://hexdocs.pm/plug/Plug.Conn.html#send_resp/3) receives `nil` when this is undefined. I'd prefer to either solve this with a static value or force the variable to be set. (Unless there's other reasoning for letting this be `nil`)

```suggestion
  version: System.get_env("SENTRY_RELEASE", "dirty"),
```

or

```suggestion
  version: System.get_env("SENTRY_RELEASE", "versioned"),
```

If we simply require this all the time now then we should do
```suggestion
  version: System.fetch_env!("SENTRY_RELEASE"),
```

Reply:
My preference is to simply always require it now, so I just pulled in your last suggestion

Reply:
Aaaand that broke the build 🤦‍♂️